bootstrap.d/app-emulation.yml

```diff
@@ -69,3 +139,89 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
+
+  - name: wine
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: Free implementation of Windows(tm) on Unix, without external patchsets
+      description: Free implementation of Windows(tm) on Unix, without external patchsets
+      spdx: 'LGPL-2.1-or-later BSD-2 IJG MIT OPENLDAP ZLIB gsm libpng2 libtiff'
+      website: 'https://www.winehq.org'
+      maintainer: 'Qwinci <qwinci222@gmail.com>'
+      categories: ['app-emulation']
+    from_source: wine
+    tools_required:
+      - host-pkg-config
+      - system-gcc
+      - virtual: pkgconfig-for-target
+        triple: '@OPTION:arch-triple@'
+      - host-wine-tools
+    pkgs_required:
+      - mlibc
+      - fontconfig
+      - freetype
+      - gnutls
+      - mesa
+      - eudev
+      - libxi
+      - libxfixes
+      - libxrender
+      - libxrandr
+      - libxshmfence
+      - libxxf86vm
+      - libjpeg-turbo
+      - libpng
+      - libxml
+      - libxslt
+      - zlib
+      # - glib
```

Comment:
What's the background for these commented-out dependencies?

Reply:
Doesn't look like they are directly needed by the latest wine so I removed them and updated the dependencies in general.